app/js/lib/server/index.js

```diff
@@ -0,0 +1,106 @@
+/* global URLSearchParams */
+
+'use strict';
+
+import EventDispatcher from '../common/event-dispatcher';
+
+import Settings from './settings';
+import Network from './network';
+import WebPush from './webpush';
+import API from './api';
+import Reminders from './reminders';
+
+// Private members.
+const p = Object.freeze({
+  // Private properties.
+  settings: Symbol('settings'),
+  net: Symbol('net'),
+  webPush: Symbol('webPush'),
+  api: Symbol('api'),
+});
+
+export default class Server extends EventDispatcher {
+  constructor({ settings, net } = {}) {
```

Comment:
I think this is mainly used for testing purposes to inject mocks into the constructor.